new-integration-tests/src/test/resources/wdt-models/model4-nginx-validation-wls.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
change file name to model-multiclusterdomain-sampleapp-wls.yaml? This model can be used for other tests if needed..

Reply:
fixed